ui/raidboss/data/05-shb/raid/e3s.js

```diff
@@ -125,9 +125,9 @@
         }
         if (data.role == 'healer') {
           return {
-            en: 'Tank Busters',
-            de: 'Tank Buster',
-            fr: 'Tank busters',
+            en: 'Tank buster',
```

Comment:
There are two tank busters, so it is Tank Buster*s* plural.

Reply:
Sorry for this topic, I remembered a discussion. #1329 For Tank Buster or Busters. that's why I changed it, but I forgot the capital letters. Everything is corrected.